### PR TITLE
Status and DEPS files updated

### PR DESCRIPTION
**Pull request by sgrekhov**
Status and DEPS files updated to use latest version of co19 tests

### CODE REVIEW

**Comment by whesse** [comment]
Committing the roll of co19 to the master branch.  This commit will be reverted immediately, after the buildbot runs have been triggered.

File: tests/co19/co19-dartium.status [hunk 1]
```diff
@@ -156,35 +140,35 @@ Language/Classes/Setters/type_object_t01: RuntimeError # Please triage this fail
...
-Language/Classes/deсlarations_t26: Skip # Times out. Please triage this failure.
-Language/Classes/deсlarations_t27: Skip # Times out. Please triage this failure.
-Language/Classes/deсlarations_t28: Skip # Times out. Please triage this failure.
-Language/Classes/deсlarations_t29: Skip # Times out. Please triage this failure.
-Language/Classes/deсlarations_t30: Skip # Times out. Please triage this failure.
-Language/Classes/deсlarations_t31: Skip # Times out. Please triage this failure.
-Language/Classes/deсlarations_t32: Skip # Times out. Please triage this failure.
-Language/Classes/deсlarations_t33: Skip # Times out. Please triage this failure.
-Language/Classes/deсlarations_t34: Skip # Times out. Please triage this failure.
+Language/Classes/declarations_t01: Skip # Times out. Please triage this failure.
```
Comment (whesse) [inline tests/co19/co19-dartium.status]: I don't see what is changed in this line.  Is there a character encoding issue?

File: tests/co19/co19-dart2js.status [hunk 2]
```diff
@@ -949,7 +970,26 @@ LibTest/typed_data/Int32x4/operator_OR_A01_t01: RuntimeError # Please triage thi
 LibTest/typed_data/Int8List/sublist_A02_t01: Pass, Slow # Please triage this failure.
 LibTest/typed_data/Uint8ClampedList/map_A02_t01: Pass, Slow # Please triage this failure.
 
+[ $compiler == dart2js && $runtime == chrome && $system != linux]
```
Comment (whesse) [inline tests/co19/co19-dart2js.status]: Please put the specific runtime == chrome sections after the general runtime == chrome section, and make sure they are merged.

File: tests/co19/co19-dart2js.status [hunk 3]
```diff
@@ -949,7 +970,26 @@ LibTest/typed_data/Int32x4/operator_OR_A01_t01: RuntimeError # Please triage thi
...
+
 [ $compiler == dart2js && $runtime == chrome ]
+LayoutTests/fast/css/getComputedStyle/getComputedStyle-zIndex-auto_t01: RuntimeError # Please triage this failure
+LayoutTests/fast/dom/shadow/link-in-shadow-tree_t01: RuntimeError # Please triage this failure
+LayoutTests/fast/forms/paste-into-textarea_t01: RuntimeError # Please triage this failure
+LayoutTests/fast/canvas/webgl/oes-vertex-array-object_t01: RuntimeError # Please triage this failure
+LayoutTests/fast/shapes/shape-outside-floats/shape-outside-floats-inset-rounded-different-writing-modes-left_t01: RuntimeError # Please triage this failure
+LayoutTests/fast/shapes/shape-outside-floats/shape-outside-rounded-boxes_t02: RuntimeError # Please triage this failure
+LayoutTests/fast/text/line-break-after-question-mark_t01: RuntimeError # Please triage this failure
+LayoutTests/fast/text/text-combine-shrink-to-fit_t01: RuntimeError # Please triage this failure
```
Comment (whesse) [inline tests/co19/co19-dart2js.status]: All lines in a section should be alphabetized (sorted alphabetically)

File: tests/co19/co19-dart2js.status [hunk 4]
```diff
@@ -1790,7 +1825,12 @@ WebPlatformTest/shadow-dom/shadow-trees/upper-boundary-encapsulation/test-009_t0
 WebPlatformTest/webstorage/event_constructor_t01: RuntimeError # Please triage this failure
 WebPlatformTest/webstorage/event_constructor_t02: RuntimeError # Please triage this failure
 
+[ $compiler == dart2js && $runtime == chrome && $checked]
```
Comment (whesse) [inline tests/co19/co19-dart2js.status]: This section seems identical to the section below.  Please merge them.

File: tests/co19/co19-dart2js.status [hunk 5]
```diff
@@ -1883,7 +1920,7 @@ LibTest/async/Timer/Timer.periodic_A01_t01: Skip # Times out. Please triage this
 LibTest/isolate/Isolate/spawn_A01_t05: Skip # Times out. Please triage this failure
 WebPlatformTest/shadow-dom/events/retargeting-focus-events/test-002_t01: Skip # Times out. Please triage this failure
 
-[ $compiler == dart2js && $runtime == chrome && $system != macos ]
+[ $compiler == dart2js && $runtime == chrome && $system != macos && $system != windows && $system != linux]
```
Comment (whesse) [inline tests/co19/co19-dart2js.status]: What system does this apply to, if it doesn't apply to macos, windows, or linux?  If these are not failing anymore, remove the section.

File: tests/co19/co19-dart2js.status [hunk 6]
```diff
@@ -1981,6 +2018,8 @@ LayoutTests/fast/canvas/webgl/webgl-layer-update_t01: RuntimeError # Please tria
 LayoutTests/fast/canvas/webgl/webgl-texture-binding-preserved_t01: RuntimeError # Please triage this failure
 LayoutTests/fast/canvas/webgl/webgl-unprefixed-context-id_t01: RuntimeError # Please triage this failure
 LayoutTests/fast/canvas/webgl/webgl-viewport-parameters-preserved_t01: RuntimeError # Please triage this failure
+
+[ $compiler == dart2js && $runtime == chrome && $system != macos && $system != windows]
```
Comment (whesse) [inline tests/co19/co19-dart2js.status]: What is the difference between this and the ==linux section immediately below?

File: tests/co19/co19-dart2js.status [hunk 7]
```diff
@@ -4134,12 +4168,60 @@ WebPlatformTest/webstorage/event_constructor_t01: RuntimeError # Please triage t
...
+LayoutTests/fast/table/table-all-rowspans-height-distribution-in-rows-except-overlapped_t01: RuntimeError # Please triage this failure
+LayoutTests/fast/table/table-colgroup-present-after-table-row_t01: RuntimeError # Please triage this failure
+LayoutTests/fast/table/table-rowspan-cell-with-empty-cell_t01: RuntimeError # Please triage this failure
+LayoutTests/fast/text-autosizing/vertical-writing-mode_t01: RuntimeError # Please triage this failure
+LayoutTests/fast/canvas/canvas-modify-emptyPath_t01: RuntimeError # Please triage this failure
+LayoutTests/fast/html/draggable_t01: RuntimeError # Please triage this failure
+LayoutTests/fast/innerHTML/005_t01: RuntimeError # Please triage this failure
+LibTest/core/RegExp/Pattern_semantics/firstMatch_Atom_A03_t03: RuntimeError # Please triage this failure
+
+[ $compiler == dart2js && $runtime == ff && $checked == true]
```
Comment (whesse) [inline tests/co19/co19-dart2js.status]: This can just be $checked, without the == true.

**Comment by whesse** [comment]
The test results are up, on https://build.chromium.org/p/client.dart/console

Click on the red squares, on the line of your commit, click on a failing step, and search for "Failed:" in your browser.  Each failure is reported twice, because all the failures are printed in a summary at the end of the output.    The test.py command line that was run to do these tests should be at the top or the bottom of the output.

Note that the tests are sharded, so there may be 4 columns running the same test suite, with each individual test just running on one of the columns.

I make notes of which tests are failing on which combination of aspects (runtime, checked mode, fast mode), so that tests which fail for all combinations just get put in the more general categories.

Try not to create new categories (sections in the status file) unless absolutely necessary.  There should be just one section for each runtime and system combination, and possibly for checked or fast mode.
Test statuses should be marked separately for each browser (firefox, chrome, ie10, ie11, etc.) unless the status applies to absolutely all browsers (in which case it probably also applies for all dart2js cases).

It would be good if you could make your changelists using the git cl upload tool, instead of as push requests, so they can be reviewed on our code review system.  See the dart-lang/sdk wiki for info on this.  But the push requests are OK, if it is too much trouble to use git cl upload.

File: tests/co19/co19-dartium.status (hunk 1 above)
Comment (sgrekhov) [inline tests/co19/co19-dartium.status]: Yes. c in word  "declaration" was not ASCII "c" character but cyrillic "с"

**Pull request merged**